### PR TITLE
added verify function deleted s character on line 10

### PR DESCRIPTION
**Pull request by wjb108**

### CODE REVIEW

File: controllers/users.js [hunk 1]
```diff
@@ -1,118 +1,128 @@
...
-        return res.status(200).json(users);
-    } catch (error) {
-        res.status(500).json({error: error.message});
-    }
+const getUsers = async (req, res) => {
+  try {
+    const users = await User.find({});
+    return res.status(200).json(users);
+  } catch (error) {
+    res.status(500).json({ error: error.message });
```
Comment (GundamMerlin) [inline controllers/users.js]: add return

File: controllers/users.js [hunk 2]
```diff
@@ -1,118 +1,128 @@
...
-    } catch (error) {
-        return res.status(500).json({error: error.message});
+const deleteUser = async (req, res) => {
+  try {
+    const { id } = req.params;
+    const deletedUser = await User.findByIdAndDelete(id);
+    if (deletedUser) {
+      return res.status(200).json({ message: "User Successfully Deleted" });
+    } else {
+      res.status(404).send("User not found");
```
Comment (GundamMerlin) [inline controllers/users.js]: add return

File: controllers/users.js (hunk 1 above)
Comment (wjb108) [reply controllers/users.js]: done

File: controllers/users.js (hunk 2 above)
Comment (wjb108) [reply controllers/users.js]: done

**Review by GundamMerlin — APPROVED** [review]

**Pull request merged**